apps/examples/log_dump/log_dump_main.c

```diff
@@ -42,13 +44,20 @@ int log_dump_main(int argc, char *argv[])
 {
 	int ret = 1;
 	char buf[READ_BUFFER_SIZE];
-	int fd = open("/proc/logsave", O_RDONLY);
+	int fd = open("/proc/logsave", O_RDWR);
 
 	if (fd < 0) {
 		printf("Please ensure that procfs is mounted\n");
 		return 0;
 	}
 
+	if (write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1) < 0)
```

Comment:
below will be more faster.
```
enum SAVE_STATUS_E {
   SAVE_STATUS_START 0,
   SAVE_STATUS_STOP 1
};

...
uint8_t status = SAVE_STATUS_START;
write(fd, &status, sizeof(status));
...
read(fd, &status, sizeof(status));
if (status == SAVE_STATUS_START) {
   do_something;
}
```
And please make sure that procfs leads to read-only file system, so if possible you had better find another way.
But if it takes too many code changes, you can use write, in my opinion.

Reply:
Yes, I'm also worried that procfs is recommended for read-only.
But for usage consistency, I implemented through write operation.
If it is not permitted to use write operation, then I will add new API for controlling the log saving functionality.
Please tell me your opinion @Taejun-Kwon .